net-analyzer/tcping/tcping-1.3.5.ebuild

```diff
@@ -1,18 +1,19 @@
-# Copyright 1999-2012 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=4
+EAPI=7
 
 inherit toolchain-funcs
 
 DESCRIPTION="Ping implementation that uses the TCP protocol"
-HOMEPAGE="http://www.linuxco.de/tcping/tcping.html"
-SRC_URI="http://www.linuxco.de/tcping/${P}.tar.gz"
+HOMEPAGE="https://github.com/mkirchner/tcping"
+SRC_URI="https://github.com/mkirchner/tcping/archive/master.tar.gz -> ${P}.tar.gz"
```

Comment:
This change is clearly invalid; it disguises a master snapshot under an existing version number, and if tomorrow there is a bump in this repo the tarball will change again.